### PR TITLE
Prow Deck: convert time to local time

### LINKED ISSUE #7016: prow: deck should localize times
I don't enjoy converting UTC to PST in my head. Prow should do that for me.

(Gubernator does this with some javascript, added in #238.)

/area prow
/assign @qhuynh96

### PR DESCRIPTION
**Pull request by qhuynh96**
* Fixes #7016 
screen shot 2018-02-27 at 11 05 11 am

### CODE REVIEW

File: prow/cmd/deck/static/moment.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
...arguments).parseZone()},e.localeData=ue,e.isDuration=Me,e.monthsShort=function(e,t){return Ee(e,t,"monthsShort")},e.weekdaysMin=function(e,t,n){return Ie(e,t,n,"weekdaysMin")},e.defineLocale=oe,e.updateLocale=function(e,t){if(null!=t){var n,s,i=Qt;null!=(s=re(e))&&(i=s._config),(n=new k(t=D(i,t))).parentLocale=Xt[e],Xt[e]=n,ae(e)}else null!=Xt[e]&&(null!=Xt[e].parentLocale?Xt[e]=Xt[e].parentLocale:null!=Xt[e]&&delete Xt[e]);return Xt[e]},e.locales=function(){return nt(Xt)},e.weekdaysShort=function(e,t,n){return Ie(e,t,n,"weekdaysShort")},e.normalizeUnits=O,e.relativeTimeRounding=function(e){return void 0===e?En:"function"==typeof e&&(En=e,!0)},e.relativeTimeThreshold=function(e,t){return void 0!==In[e]&&(void 0===t?In[e]:(In[e]=t,"s"===e&&(In.ss=t-1),!0))},e.calendarFormat=function(e,t){var n=e.diff(t,"days",!0);return n<-6?"sameElse":n<-1?"lastWeek":n<0?"lastDay":n<1?"sameDay":n<2?"nextDay":n<7?"nextWeek":"sameElse"},e.prototype=Dn,e.HTML5_FMT={DATETIME_LOCAL:"YYYY-MM-DDTHH:mm",DATETIME_LOCAL_SECONDS:"YYYY-MM-DDTHH:mm:ss",DATETIME_LOCAL_MS:"YYYY-MM-DDTHH:mm:ss.SSS",DATE:"YYYY-MM-DD",TIME:"HH:mm",TIME_SECONDS:"HH:mm:ss",TIME_MS:"HH:mm:ss.SSS",WEEK:"YYYY-[W]WW",MONTH:"YYYY-MM"},e});
```
Comment (BenTheElder) [inline prow/cmd/deck/static/moment.min.js]: License? :-)

File: prow/cmd/deck/jobs.go [hunk 2]
```diff
@@ -217,7 +217,7 @@ func (ja *JobAgent) update() error {
 			ProwJob: j.ObjectMeta.Name,
 			BuildID: buildID,
 
-			Started:     j.Status.StartTime.Format(time.Stamp),
+			Started:     string(j.Status.StartTime.Time.Unix()),
```
Comment (BenTheElder) [inline prow/cmd/deck/jobs.go]: this will break some code in the submit-queue IIRC

Comment (qhuynh96) [reply prow/cmd/deck/jobs.go]: So can I use another format layout?

Comment (BenTheElder) [reply prow/cmd/deck/jobs.go]: IIRC some code I wrote parses this to figure out which jobs ran most recently. We're phasing out the submit-queue but right now you could probably just update the submit-queue to handle whatever format you're using here.

Comment (BenTheElder) [reply prow/cmd/deck/jobs.go]: you're fine :-)
https://github.com/kubernetes/test-infra/blob/f312a14c4b9637ecd4d3804c69f5fb85e913b774/mungegithub/mungers/submit-queue.go#L743

**Comment by stevekuznetsov** [comment]
Fixes #7016

**Comment by stevekuznetsov** [comment]
Is there a simple way to have the display be "8 hours ago" and the tooltip be the actual time in your local timezone? Is that desirable? 
/cc @BenTheElder

**Comment by qhuynh96** [comment]
This is achievable and actually a good idea.

**Comment by qhuynh96** [comment]
@stevekuznetsov changed the display time to be the time from the job's start time

**Comment by stevekuznetsov** [comment]
Amazing.
/lgtm

**Comment by stevekuznetsov** [comment]
/hold

for @BenTheElder

**Comment by BenTheElder** [comment]
this is excellent looking

**Comment by ixdy** [comment]
@qhuynh96 this is awesome, thanks!

**Pull request merged**